content/en/guides/core/automations/automation-events.md

```diff
@@ -23,11 +23,9 @@ This section describes the scopes and events for an automation in a [Registry]({
 Learn more about [creating automations]({{< relref "create-automations/" >}}).
 
 ### Scopes
-You can create a Registry automation at these scopes:
-- [Registry]({{< relref "/guides/core/registry/">}}) level: The automation watches for the event taking place on any collection within a specific registry, including collections added in the future.
-- Collection level: A single collection in a specific registry.
+A [Registry]({{< relref "/guides/core/registry/">}}) automation watches for the event taking place on any collection within a specific registry, including collections added in the future.
 
-### Events
+### Events {#registry-events}
```

Comment:
Is this syntax Hugo specific? If yay, should we omit since it'll get lost in the Mintlify migration?